src/libtoast/src/toast_atm_sim.cpp

```diff
@@ -693,11 +693,9 @@ cholmod_sparse * toast::atm_sim_sqrt_sparse_covariance(
     toast::Timer timer;
     timer.start();
 
-    if (atm_verbose()) {
-        o.str("");
-        o << rank << " : Analyzing sparse covariance ... ";
-        logger.verbose(o.str().c_str());
-    }
+    o.str("");
+    o << rank << " : Analyzing sparse covariance ... ";
+    logger.verbose(o.str().c_str());
```

Comment:
The reason this guard was implemented is to avoid string manipulations / comparisons on every pass through a loop.  Instead, the logging level is checked once and a static bool is set to determine whether we need to do those string operations (only if loglevel = VERBOSE).  Could we revert this or, if we think it no longer matters, can we change all the places in the code that use this technique?

Reply:
Reverted